WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentPaymentsModalViewModel.swift

```diff
@@ -2,13 +2,12 @@ import UIKit
 
 /// Abstracts configuration and contents of the modal screens presented
 /// during operations related to Card Present Payments
-protocol CardPresentPaymentsModalViewModel {
+typealias CardPresentPaymentsModalViewModel = CardPresentPaymentsModalContent & CardPresentPaymentsModalActions
```

Comment:
💯 makes more sense to group the protocols like this and allows us to decouple UIKit

Reply:
Yeah. I'm not sure exactly how to handle the actions yet, but I'll probably add another actions protocol and make the existing view models implement it without using the viewController they otherwise get passed. We'll see.